ci-operator/step-registry/hypershift/aws/create-cluster-byoi/hypershift-aws-create-cluster-byoi-commands.sh

```
@@ -0,0 +1,106 @@
+  --pull-secret=/etc/ci-pull-credentials/.dockerconfigjson
+  --aws-creds="${AWS_GUEST_INFRA_CREDENTIALS_FILE}"
+  --release-image "${RELEASE_IMAGE}"
+  --annotations=hypershift.openshift.io/skip-release-image-validation=true
+  --additional-tags="expirationDate=$(date -d '4 hours' --iso=minutes --utc)"
+)
+
+# Execute cluster creation
+"${COMMAND[@]}"
+
+echo "Waiting for cluster to become available"
+oc wait --timeout=30m --for=condition=Available --namespace=clusters hostedcluster/${CLUSTER_NAME}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)